### PR TITLE
Tag ScikitLearn.jl v0.3.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [cstjean/ScikitLearn.jl](https://github.com/cstjean/ScikitLearn.jl)
Release: [v0.3.0](https://github.com/cstjean/ScikitLearn.jl/releases/tag/v0.3.0)
Travis: [Travis Build Status](https://travis-ci.org/cstjean/ScikitLearn.jl/builds/212141931)
Diff: [vs v0.2.4](https://github.com/cstjean/ScikitLearn.jl/compare/ca924a85bc245a00d944e95093fe0d10b58e077d...f17fd3ad01d633ab4fac3196ab4100de3f2bf45c)
`requires` vs v0.2.4: 
```diff
--- 0.2.4/requires
+++ 0.3.0/requires
@@ -1,10 +1,10 @@
-julia 0.4
+julia 0.5
 PyCall 1.6.2
 Parameters 0.3.0
 MacroTools 0.3.0
 Conda
-Requires 0.2.0
 ScikitLearnBase 0.0.6
 StatsBase 0.8.0
 Iterators 0.1.9
-Compat 0.9.2+Compat 0.17.0
+DataFrames
```
cc: @cstjean

### CODE REVIEW

File: ScikitLearn/versions/0.3.0/requires [hunk 1]
```diff
@@ -0,0 +1,10 @@
+julia 0.5
+PyCall 1.6.2
+Parameters 0.3.0
+MacroTools 0.3.0
+Conda
+ScikitLearnBase 0.0.6
+StatsBase 0.8.0
+Iterators 0.1.9
+Compat 0.17.0
```
Comment (tkelman) [inline ScikitLearn/versions/0.3.0/requires]: 0.18 for Compat.Iterators

Comment (cstjean) [inline ScikitLearn/versions/0.3.0/requires]: Fixed. I assume that I should update this PR?

Comment (tkelman) [inline ScikitLearn/versions/0.3.0/requires]: yeah, tightening a req can only be done in place, loosening could be done with a later tag

**Comment by cstjean** [comment]
Man, @attobot is cool.

**Pull request merged**